vmm/src/api/http/mod.rs

```diff
@@ -80,15 +82,28 @@ const HTTP_ROOT: &str = "/api/v1";
 /// thus insightful and helpful while balancing technical accuracy and
 /// simplicity.
 pub fn error_response(error: HttpError, status: StatusCode) -> Response {
+    const DELIMITER: &str = "<br>";
```

Comment:
I'm very happy to go ahead with merging this change as I think it is definitely a big improvement. I do wonder if we could do something like generate an JSON based response for the error instead to show the chain. ch-remote could then interpret it as you've done and also other users of the API can also get something out of it.

Reply:
Yes - this is exactly what I was thinking.

Reply:
Okay, I think I have it. In case the deserialization of the JSON itself fails (currently only can happen because of a ch-remote and Cloud Hypervisor version mismatch), it will print (note the `X` level):

```
Error: ch-remote exited with the following chain of errors:
  0: http client error
  X: Can't get remote's error messages from JSON response: EOF while parsing a value at line 1 column 0: body=''

Debug Info: HttpApiClient(ServerResponse(InternalServerError, Some("")))
```